### ISSUE TITLE
RFC: Add language as a discrimination factor

### ISSUE THREAD

**Issue by ionas** [cakephp/code-of-conduct#4]
Helloes,

language is a big problem and also a discrimination factor.
Some CoCs specify language:
- http://todogroup.org/opencodeofconduct/

I'd like to see twofold:
- An official stance on language, e.g. that all support channels, source code documentation and the book is only offering English as guaranteed but...
- That other languages are respected on XY
- That no discrimination should happen because of language barriers. As this DID IMHO happen a few month ago I see this as a case.

So there needs to be a concise sentence in essence saying: Development and communication is primarily English (practicability) but other languages are respected and discrimination because of weak English is not okay.

The implications are that the core team should not be obligated to translate docs but for instance there should be done as much as possible to include non native speakers for instance at conferences, through subtitles in videos, by welcoming their chatting in the official channel and on google groups (TBD) etc.

This is a hard issue because it is a hard problem because de facto discrimination and practicability go hand in hand.

I am not strongly pushing this, because it is hard to solve as a concise and precise don't do rule, maybe it should be a sentence in the preamble adding to the spirit of our CoC instead... but see it as a primary exclusion factor and problem.

**Comment by markstory** [comment]
I think including something about being welcoming to people who are not native english speakers and that we will do our best to support them. I think there are some limitations here though. If there is no one around who is able to answer a question in turkish (contrived example) what should we do?

**Comment by ionas** [comment]
Yes, totally on par :+1: with that. Can you think of a concise sentence that reflects that? (English is after all not my native lang).

**Comment by josegonzalez** [comment]
Something I've tried to do on IRC is ask people who appear to be struggling with english if they speak some other language natively. Sometimes we get a french or spanish speaker and I am able to better understand the intent of their question. Would be good to bring up that sort of accommodation (assuming someone who can do that is available).

Note: In some cases we _do_ have alternatives to the english-speaking communities (irc has #cakephp-br, #cakephp-fr, #cakephp-es) though I'm unaware of how well they are "staffed".

**Comment by markstory** [comment]
@ionas We could add 'proficiency or fluency with english' to the list of things to respect and be considerate about.
